### PR TITLE
test(integration): Add skip to embargo tests

### PR DESCRIPTION
**Pull request by bzang**

### CODE REVIEW

File: test/unit/spec/client/device.js [hunk 1]
```diff
@@ -213,7 +213,8 @@ describe('Client', function() {
     });
 
     describe('#refresh()', function() {
-      describe('in embargoed countries', function() {
+      // Embargo methods have changed to us geo IPs
+      describe.skip('in embargoed countries', function() {
```
Comment (ianwremmel) [inline test/unit/spec/client/device.js]: We shouldn't need to skip the unit tests.

**Comment by bzang** [comment]
Reverted @ianwremmel

**Comment by ianwremmel** [comment]
Can we close this? I think the tests are passing again.

**Comment by ianwremmel** [comment]
Closing as no longer relevant

**Pull request closed without merging**